### PR TITLE
Update reviewdog/vint versions

### PR DESCRIPTION
**Pull request by ujihisa**
* https://github.com/reviewdog/reviewdog/releases
* https://pypi.org/project/vim-vint/

### CODE REVIEW

File: scripts/run-lint-on-ci.sh [hunk 1]
```diff
@@ -7,15 +7,15 @@ fi
...
 mkdir -p ~/bin/ && export PATH="~/bin/:$PATH"
 
-# Install https://github.com/haya14busa/reviewdog
-curl -fSL \
-  https://github.com/haya14busa/reviewdog/releases/download/$REVIEWDOG_VERSION/reviewdog_linux_amd64 \
-  -o ~/bin/reviewdog && chmod +x ~/bin/reviewdog
+# Install https://github.com/reviewdog/reviewdog/releases
+curl -sfL \
+  https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh |\
+  sh -s -- -b $(go env GOPATH)/bin "v${REVIEWDOG_VERSION}"
```
Comment (haya14busa) [inline scripts/run-lint-on-ci.sh]: ~/bin

File: scripts/run-lint-on-ci.sh [hunk 2]
```diff
@@ -33,7 +33,7 @@ curl -L -o ./install-misspell.sh https://git.io/misspell \
 go get -d -v ./scripts
 
 # Run reviewdog.
-reviewdog -reporter=github-pr-check
+"$(go env GOPATH)/bin/reviewdog" -reporter=github-pr-check
```
Comment (haya14busa) [inline scripts/run-lint-on-ci.sh]: you can revert this change if you put the binary to ~/bin

Comment (ujihisa) [reply scripts/run-lint-on-ci.sh]: Thx! Fixed at 9979e57915ce28af891d04465165ca526562f450

File: scripts/run-lint-on-ci.sh [hunk 3]
```diff
@@ -33,7 +33,7 @@ curl -L -o ./install-misspell.sh https://git.io/misspell \
 go get -d -v ./scripts
 
 # Run reviewdog.
-reviewdog -reporter=github-pr-check
+~/bin/reviewdog -reporter=github-pr-check
```
Comment (haya14busa) [inline scripts/run-lint-on-ci.sh]: No need to use `~/bin`. See L13.

Comment (ujihisa) [reply scripts/run-lint-on-ci.sh]: oops right. Thx! Fixed at 30403543407a8d7882b6442370628c3ab6b60fde now.
